src/ome.ts

```diff
@@ -2,9 +2,28 @@ import { ZarrPixelSource } from '@hms-dbmi/viv';
 import pMap from 'p-map';
 import { Group as ZarrGroup, HTTPStore, openGroup, ZarrArray } from 'zarr';
 import type { ImageLayerConfig, SourceData } from './state';
-import { join, loadMultiscales, guessTileSize, range, parseMatrix } from './utils';
+import {
+  guessTileSize,
+  join,
+  loadMultiscales,
+  nested,
+  parseMatrix,
+  range
+} from './utils';
 
-export async function loadWell(config: ImageLayerConfig, grp: ZarrGroup, wellAttrs: Ome.Well): Promise<SourceData> {
+// OME-Zarr uses nested chunks since version 0.2
+function isNested(version: String | undefined) : boolean {
+  return version != undefined && version !== "0.1";
+}
+
```

Comment:
I have a slight preference to moving this function to `io.ts` and having it take the root attrs as an argument. That way we can extend this.

```typescript
function isNested(attrs: Ome.Attrs) {
  let version;
  if ('plate' in attrs) {
    version = attrs.plate.version;
  } else if ('omero' in attrs) {
    version = attrs.multiscales[0].version;
  } else if ('well' in attrs) {
    version = attrs.well.version;
  }
  return version && version !== '0.1';
}

export async function createSourceData(config: ImageLayerConfig): Promise<SourceData> {
  const node = await open(config.source);
  let data: ZarrArray[];

  if (node instanceof ZarrGroup) {
    const attrs = (await node.attrs.asObject()) as Ome.Attrs;

    if (isNested(attrs)) {
      node.store = nested(node.store);
    }

    if ('plate' in attrs) {
      return loadPlate(config, node, attrs.plate);
    }

    if ('well' in attrs) {
      return loadWell(config, node, attrs.well);
    }

    if ('omero' in attrs) {
      return loadOmeroMultiscales(config, node, attrs);
    }
/* ... */
```

Will need to repeat this logic here, but that's it:

```typescript
    if (Object.keys(attrs).length === 0 && node.store instanceof HTTPStore) {
      // No rootAttrs in this group.
      // if url is to a plate/acquisition/ check parent dir for 'plate' zattrs
      const parentUrl = node.store.url.slice(0, node.store.url.lastIndexOf('/'));
      const parent = await openGroup(new HTTPStore(parentUrl));
      const parentAttrs = (await parent.attrs.asObject()) as Ome.Attrs;
      if (isNested(parentAttrs)) {
        parent.store = nested(parent.store);
      }
      if ('plate' in parentAttrs) {
        return loadPlate(config, parent, parentAttrs.plate);
      }
    }